src/checkout_body/checkout_body.jsx

```diff
@@ -0,0 +1,248 @@
+            },
+            ]
+        },
+        topText: 'Country / Region',
+        id: 'countryes',
+        type: 'select',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: 'Street address',
+        id: 'street_adress',
+        type: 'text',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: 'Town / City',
+        id: 'town_city',
+        type: 'text',
+        placeholder: '',
+    },
+    {
+        inputType: 'select',
+        additionalInformation: {
+            countryes: [{
+                name: 'Western Province',
+                selectVal: 'west'
+            },
+            {
+                name: 'Eastern Province',
+                selectVal: 'east'
+            },
+            {
+                name: 'southern Province',
+                selectVal: 'south'
+            },
+            {
+                name: 'northern Province',
+                selectVal: 'northern'
+            },
+            ]
+        },
+        topText: 'Province',
+        id: 'countryes',
+        type: 'select',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: 'ZIP code',
+        id: 'zip',
+        type: 'text',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: 'Phone',
+        id: 'phone',
+        type: 'tel',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: 'Email address',
+        id: 'mail',
+        type: 'email',
+        placeholder: '',
+    },
+    {
+        inputType: 'standart',
+        additionalInformation: {
+        },
+        topText: '',
+        id: 'additional_inf',
+        type: 'text',
+        placeholder: 'Additional information',
+    },
+]
+
+const DoubleInput = ({ checkoutInput }) => {
+    return (
+        <div key={checkoutInput.id} className='double_iputs_container'>
+            <div className='double_iputs_item'>
+                <label className='inputs_label' htmlFor={checkoutInput.id}>{checkoutInput.topText}</label>
+                <input required className='input_text_global_pad double_checkkout_input check_input' type={checkoutInput.type} id={checkoutInput.id} />
+            </div>
+            <div className='double_iputs_item'>
+                <label className='inputs_label' htmlFor={checkoutInput.secondId}>{checkoutInput.additionalInformation.secondTopText}</label>
+                <input required className='input_text_global_pad double_checkkout_input check_input' type={checkoutInput.type} id={checkoutInput.additionalInformation.secondId} />
+            </div>
+        </div>
+    )
+}
+
+
+const StandartInput = ({ checkoutInput }) => {
```

Comment:
this is the last time I mention typo or spelling mistake. there are just too many of them. check all your files, there should be no spelling mistakes or typos anywhere. use spellchecker if you need. this just looks unprofessional
```suggestion
const StandardInput = ({ checkoutInput }) => {
```